src/js/bs4/ui.js

```diff
@@ -37,6 +37,9 @@ const dropdown = renderer.create('<div class="note-dropdown-menu dropdown-menu"
   }).join('') : options.items;
 
   $node.html(markup).attr({ 'aria-label': options.title });
+  if (options && options.codeviewButton) {
```

Comment:
Only `bs4/ui.js` has this part, not `bs3` and `lite`. Do we have to insert this on others or remove this one?

Reply:
They should have it where the button element is built, bs3 and Lite may be on different line positions. I thought I had added that and added to the PR, hence the 2 commits.

Reply:
BS3 has the code inclusion at line 176, and Lite at line 52.

Reply:
Yeah, but they only check `options.codeviewbutton` in `button` creation function. But `bs4/ui.js` have two – `button` and `dropdown` creation functions.

Reply:
oh, hm, no idea why I did that, it's only needed in the creation function, thanks for catching that.

Reply:
Do you have a better idea what we should call the `option.codeviewButton`?

Reply:
Now I'm checking the case when we apply `.note-keep` on a button which has a dropdown within it. It would be good to enable dropdown buttons automatically when the parent button has `.note-keep`class.

Reply:
ok, let me know, I'm editing the class names, and I've made the option var to be `option.codeviewKeepButton`

Reply:
Adding the option for dropdowns works, I'll add those in as well.

Reply:
<img width="363" alt="Screen Shot 2020-04-28 at 5 56 49 PM" src="https://user-images.githubusercontent.com/579366/80468076-ade90880-8979-11ea-831e-348eba8759e7.png">
I concluded it would not be good in some cases — not everyone wants to enable all buttons within a dropdown automatically. So let's keep it as simple as possible and let users can choose one by one. Above picture shows only one button in a dropdown is enabled while in codeview.

Reply:
oh, yeah, that is an issue, I figured it would be more for plugins anyway, as the default buttons wouldn't be needed in codeview.